ludwig/api.py

```diff
@@ -162,8 +162,13 @@ def train(
             Intermediate preprocess structure containing the mappings of the input
             CSV created the first time a CSV file is used in the same
             directory with the same name and a '.json' extension.
-        :param data_format: (str, default: `None`) format to interpret data sources. Will be inferred
-               automatically if not specified.
+        :param data_format: (str, default: `None`) format to interpret data
```

Comment:
I know I'm being pedantic... but I would put backticks before each format in the list, so that the commas are outside of the backticked part

Reply:
np...I actually thought that but decided to be lazy...got caught by the professor. :-)
done.